setup.py

```diff
@@ -136,21 +136,24 @@ def dirty_version():
 ORIGINAL_VERSION_LINE = None
 
 
-def replace_version(base, N):
+def replace_version(N):
     """Replace version in `__init__.py` with devN suffix"""
     global ORIGINAL_VERSION_LINE
     with open('xonsh/__init__.py', 'r') as f:
         raw = f.read()
     lines = raw.splitlines()
+    assert '__version__' in lines[0], 'version should define at the top'
```

Comment:
Please make this message more explicit about the `__init__.py` file, for example `'__version__' must be the first line of the __init__.py`